packages/explorer/src/consts.ts

```diff
@@ -0,0 +1,28 @@
+import { Hex } from "viem";
+import { privateKeyToAccount } from "viem/accounts";
+
+export const NON_EDITABLE_TABLES = ["__chainState", "__mudStoreTables"];
```

Comment:
we could prob import this from store-sync/sqlite so we don't have to hardcode here

Reply:
I've imported the names from `store-sync`, although it's pretty horrible due to how types are created in `drizzle-orm`. Will try to check again if I can come up with anything better